### PR TITLE
Switch XCB to Xlib

### PR DESCRIPTION
**Pull request by HidenoriMatsubayashi**
Window resize with OpenGL/EGL doesn't work properly when using XCB. So, I changed XCB to Xlib to resolve this problem.

## Related issues
- https://github.com/sony/flutter-embedded-linux/issues/37

### CODE REVIEW

**Comment by HidenoriMatsubayashi** [comment]
@Mai-Matsuura Will you review this PR?

**Review by Mai-Matsuura — COMMENTED** [review]
Please check my comment.

File: src/flutter/shell/platform/linux_embedded/window/native_window_x11.cc [hunk 1]
```diff
@@ -14,75 +15,61 @@
...
-    LINUXES_LOG(ERROR) << "Failed to get xcb connection.";
-    return;
+NativeWindowX11::NativeWindowX11(Display* display, VisualID visual_id,
+                                 const size_t width, const size_t height) {
+  XVisualInfo visualTemplate;
+  visualTemplate.visualid = visual_id;
+
+  int visualsCount;
+  auto* visual =
+      XGetVisualInfo(display, VisualIDMask, &visualTemplate, &visualsCount);
```
[the remark this answers is not in the document]
Comment (HidenoriMatsubayashi) [reply src/flutter/shell/platform/linux_embedded/window/native_window_x11.cc:29]: Yes, it does. I've added the error handling.

**Comment by Mai-Matsuura** [comment]
@HidenoriMatsubayashi I've checked the changes and it looks fine.

**Pull request merged**